php/php.editor/src/org/netbeans/modules/php/editor/parser/astnodes/NamespaceName.java

```diff
@@ -76,7 +76,7 @@ public static NamespaceName create(int start, int end, @NonNull String namespace
         boolean isCurrent = namespaceName.startsWith("namespace\\"); // NOI18N
         String[] names = namespaceName.split("\\\\"); // NOI18N
         int startSegment = start;
-        List<Identifier> list = new ArrayList<>();
+        List<Identifier> list = new ArrayList<>(names.length);
```

Comment:
Fixed this: https://github.com/apache/netbeans/pull/4037#pullrequestreview-982945274